app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -288,16 +288,29 @@ class NavigationOverlayFragment : Fragment() {
                         fxaButton.contentDescription =
                             resources.getString(R.string.fxa_navigation_item_new,
                                 resources.getString(R.string.app_name))
+                        resetFxaOnboardingShown()
```

Comment:
fwiw, I think a slightly simpler approach would be to launch the onboarding from [the login success callback](https://searchfox.org/mozilla-mobile/rev/84667ea7969a4c48afba2f9a17e0b2940a33db01/firefox-tv/app/src/main/java/org/mozilla/tv/firefox/fxa/FxaLoginUseCase.kt#116) – so we don't need to keep track of a boolean – but perhaps it's not worth changing now.